spec/mailers/comment_mailer_spec.rb

```diff
@@ -2,6 +2,8 @@
   include MailerSpecHelper
   include EnvVarSpecHelper
 
+  let(:user) { create(:user) }
```

Comment:
I wish we could.

however, the rspec retry approach to workaround DeadClient failures has caching problems with our handrolled memoization when we use a `def` method.  see https://github.com/18F/C2/pull/1099#issuecomment-201404576